Src/FluentAssertions/Execution/AssertionScope.cs

```diff
@@ -72,17 +73,26 @@ public AssertionScope()
         /// <summary>
         /// Starts a named scope within which multiple assertions can be executed and which will not throw until the scope is disposed.
         /// </summary>
-        public AssertionScope(string context)
+        public AssertionScope(Lazy<string> context)
```

Comment:
The `string` constructor should be kept
Just add a new constructor instead.

Reply:
:+1: